### PR TITLE
fix(learn): React propTypes challenge, improve test text

### LINKED ISSUE #39204: React PropTypes Challenge Unclear
https://www.freecodecamp.org/learn/front-end-libraries/react/use-proptypes-to-define-the-props-you-expect

In the instructions and unit tests are ambiguous with the word "required".

It states:
"The Items component should include a propTypes check that requires quantity to be a number."

This is unclear as "requires" could mean that this check just simply needs to exist for the test to pass or that quantity should be a number that is required. 

I suggest either changing the above wording, or adding an additional unit test:
"The Items component should include a quantity propType that is required."

### PR DESCRIPTION
**Pull request by walterpalagi**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39204

See RandellDawson's [request](https://github.com/freeCodeCamp/freeCodeCamp/issues/39204#issuecomment-698545162).

### CODE REVIEW

File: curriculum/challenges/english/03-front-end-libraries/react/use-proptypes-to-define-the-props-you-expect.english.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ tests:
     testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(ShoppingCart)); return mockedComponent.find('ShoppingCart').length === 1; })());
   - text: The <code>Items</code> component should render.
     testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(ShoppingCart)); return mockedComponent.find('Items').length === 1; })());
-  - text: The <code>Items</code> component should include a <code>propTypes</code> check that requires <code>quantity</code> to be a <code>number</code>.
+  - text: The <code>Items</code> component should include a <code>propTypes</code> check to require a value for <code>quantity</code> and ensure that its value is a <code>number</code>.
```
Comment (RandellDawson) [inline curriculum/challenges/english/03-front-end-libraries/react/use-proptypes-to-define-the-props-you-expect.english.md:34]: "number" should not have code tags around it.
```suggestion
  - text: The <code>Items</code> component should include a <code>propTypes</code> check to require a value for <code>quantity</code> and ensure that its value is a number.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@walterpalagi Thanks for the PR.  I have made a small suggestion.  Since number is not a value or a variable name, the code tags are not needed. If you agree, you can simply click the `Commit suggestion` button and create a custom commit message like `fix: removed code tags around number`.

**Comment by walterpalagi** [comment]
Done!

**Review by RandellDawson — APPROVED** [review]
LGTM

**Review by naomi-lgbt — APPROVED** [review]
LGTM! 🥳

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**